### ISSUE TITLE
UX: Confusion over issues for files on the upload page

### ISSUE THREAD

**Issue by monkeyiq** [filesender/filesender#368]
When dropping files/folders if the file has an issue such as it is empty or we have gone beyond the max file limit, the error does not show in a way that users see easily by default. For example dropping in 100 files the user needs to scroll through the list to see that a file is red. Maybe a popup when clicking send should alert the user or something along those lines?

This issue was reported by Michael D’Silva

**Comment by glbrimhall** [comment]
I ran into this during the early stages of development of directory upload, and a suggestion I posted was to sort all the error files to the top of the file list after a drag-n-drop so that way they are conspicous, relatively easy to access since they are at the top, and can get dealt with on a case-by-case bases by the user.

At the time this wasn't my main priority but at this point I'm glad people are seeing the need for some UX solution for dealing with error files during directory upload.

I'll wait a few days to see what other UX ideas come up before diving into implementation where error files are auto-sorted to the top (a pure client side javascript fix).

One Q: There may be a need to support uploading empty files ( ie file exists, but length is zero ). I'm thinking  this should get reported and dealt with as a seperate issue than this UX one.
